.changelog.md.twig

```diff
@@ -6,27 +6,39 @@ This release includes CMS and Framework version X.X.X.
 
 - [Framework X.X.X](#)
 
-Upgrading to Recipe {{ version }} is optional, but is recommended for all CWP sites.
+Upgrading to Recipe {{ version }} is recommended for all CWP sites. This upgrade can be carried out by any development team familiar with SiliverStripe CMS. However, if you would like SilverStripe's assistance, you can request support via the [Service Desk](https://www.cwp.govt.nz/service-desk/new-request/).
 
-This upgrade can be carried out by any development team familiar with SilverStripe CMS, but if you would like SilverStripe's assistance, please let us know.
+## New Features
 
-## Upgrading instructions
+The [release announcement](#) includes the note worthy features, but be sure to review the change log for full detail.
 
-In order to update an existing site to use the new basic recipe the following changes to your composer.json can be made:
+{{ other upgrade notes here }}
 
-```
-...
-```
+## Known Issues
+
+* {{ [Known issue](GitHub link) }}
+
+### Expected test failures
+
+The following PHPUnit test failures are expected and do not represent functional issues in CWP:
+
+* {{ [test failure] }}
```

Comment:
FYI the `{{ abc }}` syntax has special meaning as a placeholder in Twig templates. I suspect it won't matter unless we define a replacement for them though. May be best to use something else like `*` or `_` anyway

Reply:
Thanks for the heads up. Updated to use Twig comment syntax, see https://twig.symfony.com/doc/2.x/templates.html#comments